benchmarking/smartseq2/merge_expression_matrix/run_merge_quants.wdl

```diff
@@ -0,0 +1,40 @@
+task mergeTPM{
+  String target_dir
+  String run_name
+  String output_name
+  String uuid
+  String value_name
+  command{
+    gsutil cp -r ${target_dir} ./   
+    python pipeline_testing_scripts/merge_table_quants.py  ${uuid} ${run_name} ${value_name} ${output_name}
+  }
+
+  output{
+    File merged_tpm_csv = "${output_name}"
+  }
+  runtime{
+    docker:"gcr.io/broad-dsde-mint-dev/analysis-tools:0.0.3"
+    memory:"7.5 GB"
+    disks: "local-disk 100 HDD"
+  }
+}
+
+workflow run_merge{
+  String scripts_dir
+  String uuid
```

Comment:
I think it will be more convenient for automation etc if run_merge takes an Array[File] instead of a uuid that we use to get the metadata and then manually download each file.

I would see us running an outer wdl that calls the ss2 pipeline, then runs the two run_merge workflows. That way, the outputs from the ss2 pipeline can be passed directly to the merge workflows without needing to use requests.get to download the metadata (which also requires us to have the username and password available, and requires the docker image to be private, which complicates things) and then download each file.

Merging single cell metrics files seems like a pretty generic task that should go into a Python library, for example sctools. One advantage of putting it in a library like sctools is that we could then have unit tests for it that trigger on every PR. It would also make it a little easier for ourselves and others to install and use with pip install (for us we would pip install into a docker image).

@ambrosejcarr -- any reason why we shouldn't put code to merge quantification and metrics files into sctools? 

There may be opportunities to make some of the code more generic here, but in principle code to merge files of 
these types seems within the realm of sctools to me. I think it's best to consolidate our Python code for working with single cell data into one library, unless there's a really good reason why we need to split it up.

I think the WDLs also make sense to go into skylab in the test directory (understanding the term "test" in a broader sense to mean characterizing pipelines). That's where I think people would tend to look for it. Or perhaps in a benchmarking directory, although I think putting it in test makes things simpler.